tests/system/feature_store/test_spark_engine.py

```diff
@@ -1524,6 +1524,75 @@ def test_ingest_and_get_offline_features_with_hdfs(self):
 
         resp_df.reset_index(drop=True, inplace=True)
         pd.testing.assert_frame_equal(resp_df[["bad", "department"]], expected_df)
+        target.purge()
+        with pytest.raises(FileNotFoundError):
+            target.as_df()
+        target.purge()
+
+    @pytest.mark.skipif(
+        not {"HDFS_HOST", "HDFS_PORT", "HDFS_HTTP_PORT", "HADOOP_USER_NAME"}.issubset(
+            os.environ.keys()
+        ),
+        reason="HDFS host, ports and user name are not defined",
+    )
+    def test_hdfs_wrong_credentials(self):
+        host = os.environ.pop("HDFS_HOST")
+        try:
+            datastore_profile = DatastoreProfileHdfs(
+                name="my-hdfs",
+                host="localhost",
```

Comment:
Looks like this is a test for a wrong host, not wrong credentials.

Reply:
changed to user mistake